scripts/gerrit_jenkins/parse_output_parser.py

```diff
@@ -0,0 +1,39 @@
+import fileinput
+import json
+import re
+
+def parse_issue(issue, review):
+  start = '/var/jenkins_home/workspace/cc-clangsa_checkers/clangsa_checkers/'
+  line = issue[0]
+  rel_file_path = line[line.find(start) + len(start) : line.find(':')]
+  splitted = line.split(':')
+  line_num = splitted[1]
+  column_num = splitted[2]
+  if rel_file_path not in review['comments']:
+    review['comments'][rel_file_path] = []
+  review['comments'][rel_file_path].append(
+    {"range":{ "start_line": line_num,
+               "start_character": column_num,
+               "end_line": line_num,
+               "end_character": column_num}, "message": ''.join(issue)})
+
+read = False
+issues = []
+for line in fileinput.input():
+  r = re.compile("^\[.*\]")
+  if re.match(r, line):
+    read = True
+    issues.append([line])
+  elif line == '\n':
+    read = False
+  elif read:
+    issues[-1].append(line)
+
+result = 1 if len(issues) == 0 else -1
+review = {"tag": "jenkins", "message":
+            "CodeChecker found %i issue(s) in the code" % len(issues),
+          "labels": {"Code-Review": result, "Verified": 1}, "comments": {}}
+for issue in issues:
+  parse_issue(issue, review)
+
+print json.dumps(review)
```

Comment:
Please use python3 compatible print statement `print()`
could we use just `json.dump` to `sys.stdout`?